src/finufft_core.cpp

```diff
@@ -85,6 +78,8 @@ Design notes for guru interface implementation:
 namespace finufft {
 namespace common {
 
+static constexpr double PI = 3.14159265358979329;
```

Comment:
Shall we just make a namespace for all our constants?

Reply:
Maybe not right now :)
But I just committed a new `simpleinterfaces.cpp` using your type names, and it looks much better indeed. Given that I added the "complicated interfaces" to this file as well, it really hasn't become much longer than before...
Let me know what you think; it's easy to revert this.